src/style.css

```diff
@@ -1 +1,68 @@
 /* Styles go here */
+
+body {
+  margin: 0;
+}
+
+.header__wrap {
+  display: flex;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  height: 60px;
+}
+
+.li__link {
+  text-decoration: none;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  color: #000;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  margin-right: 20px;
+}
+
+.ul__wrap {
+  display: flex;
+  padding: 0;
+  align-items: center;
+  text-transform: uppercase;
+}
+
+.li__wrap {
+  list-style: none;
+
+}
+
+.stop {
+  margin: 0;
+}
```

Comment:
no no, don't use classes like these when you have perfect `:last-child` pseudoclass
```suggestion
```